src/style.css

```
@@ -0,0 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 198px;
+  height: 406px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  color: #616070;
```

```suggestion
  color: #616070;
  padding: 32px 16px 16px;
```
set paddings for card and remove same margins in other elements